src/test/config/bitbucket/helm_parameters

```diff
@@ -11,7 +11,7 @@ ELASTICSEARCH_APP_VERSION=7.16.3
 LOG_DOWNLOAD_DIR=${project.build.directory}/logs
 TARGET_NAMESPACE=${kubernetes.target.namespace}
 DOCKER_IMAGE_REGISTRY=${dockerImage.registry}
-DOCKER_IMAGE_VERSION=${dockerImage.version}
+DOCKER_IMAGE_VERSION=8.6.1
```

Comment:
Override default version to test mesh

Reply:
Should we change this when the new version become the default one? In this case is good to document this somewhere to not forgotten when the default version changed in future.